docs/releases/2.2.txt

```diff
@@ -197,6 +197,8 @@ Migrations
 
 * ``NoneType`` can now be serialized in migrations.
 
+*  Allowed registering serializers with ``MigrationWriter``.
```

Comment:
Yes, that's what I suggested. A release note for an undocumented feature doesn't make sense.

Reply:
can i write the doc this part

```
diff --git a/docs/topics/migrations.txt b/docs/topics/migrations.txt
index 3f0a1fa68c..f5433b93c0 100644
--- a/docs/topics/migrations.txt
+++ b/docs/topics/migrations.txt
@@ -750,6 +750,10 @@ The decorator adds logic to capture and preserve the arguments on their
 way into your constructor, and then returns those arguments exactly when
 deconstruct() is called.

+Custom Serializers
+---------------------------------
+Django allows custom serializer methods since version 2.2. You can create a serializer ...
+
 Supporting multiple Django versions
 ===================================
```